### DISCUSSION TITLE
Is there a reason auto buy is not implemented?

### DISCUSSION THREAD

**Discussion by pacafuentes** [jef/streetmerchant#2191] [Help]
Hi! I'm surprise with the amount of features this project have, is very complete. So I imagine there is a limitation somewhere to prevent the auto buy feature. Which is the limitation?

Thanks!!

**Comment by 0000day** [comment]
Its the same reason why the banana is bend. Because no one came to do it. Clone the repo and implement the feature. You'll be a hero ^^.
